src/input/mod.rs

```diff
@@ -1649,6 +1707,16 @@ impl State {
             return FilterResult::Intercept(None);
         }
 
+        // TODO modifiers queue
```

Comment:
Is this todo problematic? Can this currently interact weirdly with modifier only keybindings? (mostly for future reference to understand what needs to be done here.)

Reply:
I think that was just a comment to remind myself to look into whether this interacts problematically with that at some point. Not for any known issue.

I'll just remove that comment, but more testing could definitely be done in general.